UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/BannerNotification/index.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿
+<!--Phase Banner-->
+ @if (!string.IsNullOrEmpty(ViewBag.BannerNotificationMessage))
+ {
+    <div class="phase-banner">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-sm-12">
+                    <P>
+                        @Html.Raw(ViewBag.BannerNotificationMessage)
```

Comment:
Please validate and prevent XSS before rendering html for banner notification message.

Reply:
Done required changes added HtmlSanitizer NuGet package to prevent XSS before rendering html.

Reply:
Good to see, this was a concern of mine coming out of the Show-and-Tell